pkg/core/rest/errors/error_handler.go

```diff
@@ -39,6 +39,12 @@ func HandleError(ctx context.Context, response *restful.Response, err error, tit
 			Title:  "Bad Request",
 			Detail: err.Error(),
 		}
+	case errors.Is(err, &BadRequest{}):
+		kumaErr = &types.Error{
+			Status: 400,
+			Title:  title,
+			Detail: err.Error(),
+		}
```

Comment:
Should you should have this another condition in the previous case?

Reply:
Thought about this bad then we won't have title configurable if we are ok with this